Samsung/z3s/res/values/config.xml

```diff
@@ -0,0 +1,256 @@
+<resources>
+    <string name="config_cameraLaunchGestureSensorStringType"/>
+    <string name="config_mainBuiltInDisplayCutout"/>
```

Comment:
uh? S20 Ultra definitely has a cutout

Reply:
It from strings.xml as below. I have run the test.sh and no fatal error exits.
<img width="682" alt="image" src="https://user-images.githubusercontent.com/29745604/177061474-50f5bd7d-b6c0-4f91-be05-4fa4565bf29b.png">